src/orquestra/sdk/_base/cli/_ui/_presenters.py

```diff
@@ -362,19 +362,51 @@ def show_wf_run(self, summary: ui_models.WFRunSummary):
         self._console.print(self.get_wf_run(summary))
 
     def show_wf_list(self, summary: ui_models.WFList):
+        """Display a list of workflow runs.
+
+        The list is shown as a table with the following headings:
+        - Workflow Run ID
+        - Status
+        - Succeeded Tasks
+        - Start Time
+        - [Owner]
+
+        The 'Owner' column is displayed as long as at least one run in the list has a
+        populated ``owner`` field.
+
+        Example output::
+
+            Workflow Run ID   Status     Succeeded Tasks   Start Time                 Owner
+            ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+            wf1               SUCCEEDED  1/1               Fri Feb 24 08:26:07 2023   evil/emiliano.zapata@zapatacomputing…
+            wf2               WAITING    0/1                                          evil/emiliano.zapata@zapatacomputing…
+
+        Args:
+            summary: A list of workflow run summaries to be displayed.
+        """  # noqa: E501
+        show_owner: bool = False
+        for row in summary.wf_rows:
+            if row.owner:
+                show_owner = True
+                break
+
         table = Table(
             "Workflow Run ID",
             "Status",
             "Succeeded Tasks",
             "Start Time",
             box=SIMPLE_HEAVY,
         )
+        if show_owner:
+            table.add_column(header="Owner")
+
         for run in summary.wf_rows:
             table.add_row(
                 run.workflow_run_id,
                 run.status,
                 run.tasks_succeeded,
                 _format_datetime(run.start_time),
+                run.owner,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏If we don't show owners, what happens if we add extra column?